packages/handleRichTextBoxKeyEventNavigation/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "quip-apps-handle-richtextbox-key-event-navigation",
-    "version": "0.1.2-alpha.17",
+    "version": "0.1.2-alpha.18",
```

Comment:
These are bumped programatically on merge, so no need to update here (it may even cause issues to do any manual versioning, [lerna](https://lerna.js.org/) handles this for us.)